server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/delivery/LocalDelivery.java

```diff
@@ -118,7 +118,6 @@ public void init(MailetConfig mailetConfig) throws MessagingException {
             .userRepository(usersRepository)
             .resourceLocator(ResourceLocatorImpl.instanciate(usersRepository, sieveRepository))
             .consume(getInitParameter("consume", true))
-            .deliveryHeader("Delivered-To")
```

Comment:
There is no configuration entry.
I don't think it is needed as one can use AddHeader mailet if he really wants.

(It turns out the Delivered-To header is not that conventionnal)